### DISCUSSION TITLE
remove "workspace" from detail displayed in windowcd?

### DISCUSSION THREAD

**Discussion by pbwn** [davatorium/rofi#1910] [General]
hello. The documentation says windowcd "shows a list of windows on the current desktop". I wonder why this list displays the workspace then, i.e. if it is the current desktop it must be showing the same workspace for every item in the list and is therefore superfluous? In any case, i don't use workspaces at all and would love to suppress this detail. Is it possible? I have read the man page and anything else i can find with searx (not great search) and cannot see if it's possible or how to go about it. Thanks for developing rofi!

**Comment by DaveDavenport** [comment]
Yes, you can. See [manpage](https://davatorium.github.io/rofi/1.7.5/rofi.1/#window-switcher-settings).

**Reply by pbwn** [reply]
thanks, i've gone thru it several times though. Can you point me to a section? cheers

**Reply by pbwn** [reply]
i just went thru it again and don't see it ...

**Reply by DaveDavenport** [reply]
Its the first config option the link refers too?

**Reply by DaveDavenport** [reply]
`rofi -window-format '{t}' -show windowcd `

**Reply by pbwn** [reply]
excellent, many thanks. I would support $ but it doesn't seem possible
